docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -38,95 +77,117 @@ As for the domain URL, Feel free to get creative — just remember that the URL
+- Next, change the `Application code` setting to `Upload your code`
 
-### Base Configuration
+![Elastic Beanstalk Base Configuration](images/EBUploadYourCode.png)
 
-![Elastic Beanstalk Base Configuration](images/EBBaseConfig.png)
+- On the Source Code Origin section click the `Choose file` button with the `Local File` radio button selected and upload the file you dowloaded on the very beginning of this guide (metabase-aws-eb.zip):
 
-While most of the fields here will be correctly pre-filled by following the launch URL above, you'll just need to do two things:
+![Elastic Beanstalk Base Configuration](images/EBUploadZipFile.png)
 
-- First, make sure `Platform` is set to `Preconfigured platform`, with the dropdown set to `Docker`.
-- Next, change the `Application code` setting to `Upload your code`, then click the `Upload` button. You'll see a modal like this:
+These settings will run the Metabase application using [Docker](https://www.docker.com) under the hood, using the official Metabase Docker image which is [published on Dockerhub](https://hub.docker.com/r/metabase/metabase/).
 
-![Elastic Beanstalk Base Configuration](images/EBUploadYourCode.png)
+You can now go ahead and click `Review and launch` and you will be directed to a page with many sections where you will have to do a bit more configuration to launch your instance.
 
-The modal should be pre-populated with the correct settings if you used the quick launch URL at the top of these instructions. All you need to do is push the `Upload` button in this modal.
+## Step 2 - Configure the basic Metabase architecture
 
-These settings will run the Metabase application using [Docker](https://www.docker.com) under the hood, using the official Metabase Docker image which is [published on Dockerhub](https://hub.docker.com/r/metabase/metabase/).
+### 2.1 Enabling enhanced health checks
 
-You can now go ahead and click `Review and launch`.
+You will need to enable enhanced health checks for your Elastic Beanstalk environment as this will tell the load balancer that the environment is healthy.
 
-#### Enabling enhanced health checks
+Click on the `Edit` link under the Load Balancer section as seen here:
 
-You will need to enable enhanced health checks for your Beanstalk Monitoring.
+![Elastic Beanstalk Monitoring](images/EBLoadBalancerEdit.png)
 
-Click on the `Modify` link under the Monitoring section as seen here:
+In the Processes section, select the default process and click on `Actions` -> Edit
 
-![Elastic Beanstalk Monitoring](images/EBMonitoringSelect.png)
+![Elastic Beanstalk Monitoring Process](images/EBProcessesSection.png)
 
-The `Health check path` is how Elastic Beanstalk knows when the application is ready to run. This path is also used by the Load Balancer to check application heath. Set this path to `/api/health`.
+The `Health check path` is where the Load balancer asks the application if its healthy so it can send traffic to. Set this path to `/api/health`
```

Comment:
```suggestion
The **Health check path** is where the Load balancer asks the application if its healthy so it can send traffic to. Set this path to `/api/health`
```